styles/components/progressbar.module.css

```diff
@@ -0,0 +1,39 @@
+.container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    position: relative;
+}
+
+.base_line {
+    position: absolute;
+    height: 3px;
+    background-color: #EAE0D5;
```

Comment:
not imp but i think we can use the css variable we have in global css config. Might be useful if we ever want to change themes. your call tho since this is not blocking feature so can be done later

Reply:
Agree